index.html

```diff
@@ -11,13 +11,13 @@
       UPLOADCARE_PUBLIC_KEY = 'eb0df3191b88f403ad76';
     </script>
     <script src="https://ucarecdn.com/widget/2.5.5/uploadcare/uploadcare.full.min.js"></script>
-    <script src="https://cdn.rawgit.com/zenorocha/clipboard.js/v1.5.12/dist/clipboard.min.js"></script>
-    <script src="vendor/aframe-master.min.js"></script>
+    <script src="https://aframe.io/releases/1.3.0/aframe.min.js"></script>
     <script src="vendor/aframe-input-mapping-component.js"></script>
     <script src="build.js"></script>
-    <script src="https://unpkg.com/aframe-gltf-exporter-component/dist/aframe-gltf-exporter-component.min.js"></script>
-    <script src="vendor/aframe-teleport-controls.min.js"></script>
-    <script src="vendor/aframe-tooltip-component.min.js"></script>
+    <script src="https://cdn.jsdelivr.net/npm/clipboard@1.5.12/dist/clipboard.min.js"></script>
```

Comment:
What is clipboard used for?

Reply:
For the Copy URL button when you upload a painting